hatch.toml

```diff
@@ -44,6 +44,8 @@ fmt = [
     "black {args:.}",
     "ruff --fix {args:.}",
     "style",
+    "typing",
+    "spelling",
```

Comment:
We could also add the following hatch scripts to the list:`riot`, `security`, `test`, and `suitespec-check`